hs_access_control/management/utilities.py

```diff
@@ -0,0 +1,73 @@
+"""
+Utility functions for management commands for access control.
+"""
+
+from django.contrib.auth.models import User, Group
+from hs_access_control.models.community import Community
+import re
+
+
+def group_from_name_or_id(gname):
+    """ return a group object given either an id or a name """
+
+    RE_INT = re.compile(r'^([1-9]\d*|0)$')
```

Comment:
move out of the method so it's only compiled once?

Reply:
Done.